app/controllers/api/v2/discovery_rules_controller.rb

```diff
@@ -29,9 +29,9 @@ def show
       def_param_group :discovery_rule do
         param :discovery_rule, Hash, :action_aware => true do
           param :name, String, :required => true
-          param :search, String, :required => true
-          param :hostgroup_id, Integer, :required => true
-          param :hostname, String, :required => true
+          param :search, String, :required => true, :desc => N_("query to match discovered hosts for the particular rule.")
+          param :hostgroup_id, Integer, :required => true, :desc => N_("puts the rules in order, low numbers go first. Must be greater then zero.")
+          param :hostname, String
           param :max_count, Integer
           param :priority, Integer
           param :enabled, :bool
```

Comment:
Can you add descriptions for all of them? It's all documented. Hostname is also interesting, max_count as well...